docs/src/development/environment-setup.md

```diff
@@ -82,3 +79,7 @@ Some typos can be automatically fixed by running
 ```shell
 $ typos -w
 ```
+
+## Contributing
+
+Read the general contribution guideline [here](https://github.com/foundry-rs/starknet-foundry/blob/master/CONTRIBUTING.md)
```

Comment:
Would be nice to update this file too I guess

Reply:
Done, please lmk @piotmag769 @Arcticae if you think we should add there something.